mongodb/spring-cloud-starter-stream-source-mongodb/src/main/java/org/springframework/cloud/stream/app/mongodb/source/MongoDBSourceApplication.java

```diff
@@ -0,0 +1,18 @@
+package org.springframework.cloud.stream.app.mongodb.source;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.SpringBootApplication;;
+
+@SpringBootApplication
+@EnableAutoConfiguration
+public class MongoDBSourceApplication {
+
+    protected final Logger LOG = LoggerFactory.getLogger(MongoDBSourceConfiguration.class);
+
+    public static void main(String[] args) {
+        SpringApplication.run(MongoDBSourceApplication.class, args);
+    }
+}
```

Comment:
There is not a need for having the main application class here. Add a new tag here for generating the app. https://github.com/spring-cloud/spring-cloud-stream-app-starters/blob/master/spring-cloud-stream-app-generator/pom.xml

Then you can generate the app (instructions are in the main doc).